pkg/utils/build/custom_bundle.go

```diff
@@ -23,20 +24,29 @@ const (
 	testBundle   = "quay.io/konflux-ci/tekton-catalog/task-buildah-min:0.4"
 )
 
-func GetDockerBuildPipelineBundle() map[string]string {
-	var buildPipelineAnnotation map[string]string
-	customDockerBuildBundle := os.Getenv(constants.CUSTOM_DOCKER_BUILD_PIPELINE_BUNDLE_ENV)
-	if customDockerBuildBundle != "" {
-		buildPipelineAnnotation = map[string]string{
-			"build.appstudio.openshift.io/pipeline": fmt.Sprintf(`{"name":"docker-build", "bundle": "%s"}`, customDockerBuildBundle),
-		}
-	} else {
-		buildPipelineAnnotation = constants.DefaultDockerBuildPipelineBundle
+func GetDockerBuildPipelineBundleAnnotation(buildPipelineName constants.BuildPipelineType) map[string]string {
```

Comment:
nit pick: will it be good to rename it `GetBuildPipelineBundleAnnotation`, sounds better to me